src/main.cpp

```diff
@@ -74,6 +78,7 @@ int main ( int argc, char** argv )
     bool         bCustomPortNumberGiven      = false;
     int          iNumServerChannels          = DEFAULT_USED_NUM_CHANNELS;
     quint16      iPortNumber                 = DEFAULT_PORT_NUMBER;
+    int          iJsonRpcPortNumber          = -1;
```

Comment:
Tiny comment: I'm not keen on this being a literal rather than (a new) `INVALID_PORT` define (or some pre-existing invalid port define that I'm not aware of).  I guess I have to live with it being a datatype bigger than the valid port number range, of course, so -1 is probably the correct value to use, in case that range ever gets bigger.

Reply:
I will check if there is such define, and if there isn't one, I will add `UNSPECIFIED_PORT_NUMBER`.

Reply:
> I will add `UNSPECIFIED_PORT_NUMBER`.

I'd rather `INVALID_PORT`.  It doesn't need "number" and it should be a value that is definitely invalid.  Zero really means "unspecified, do what you like" as a port number, so saying "unspecified" isn't clear.

Reply:
`INVALID_PORT` added.